exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = true; // finish this statement
```

Comment:
You need to assign to comparison expression not boolean so expression have to return boolean value. And all other expression have to change too.

Reply:
Thanks for your feedback Mustafa.I will do it.